src/analyses/ai.cpp

```diff
@@ -506,7 +506,7 @@ bool ai_baset::do_function_call_rec(
     // We can't really do this here -- we rely on
     // these being removed by some previous analysis.
   }
-  else if(function.id()=="NULL-object")
+  else if(function.id()==ID_null_object)
```

Comment:
Nit pick (here and almost everywhere else in this PR): the clang-format Travis job has some suggestions to introduce whitespace around `==`. I'd follow that, but that's nit picking obviously.